File: Python/Colleen.py
# Print function

def print_f(s):
	print(s % (10, 10, 10, 9, 37, 10, 10, 10, 9, 10, 9, 34, s, 34, 10, 9, 10, 10, 34, 34, 10, 9))

def main():
	# Main function
	s = "# Print function%c%cdef print_f(s):%c%cprint(s %c (10, 10, 10, 9, 37, 10, 10, 10, 9, 10, 9, 34, s, 34, 10, 9, 10, 10, 34, 34, 10, 9))%c%cdef main():%c%c# Main function%c%cs = %c%s%c%c%cprint_f(s)%c%cif __name__ == %c__main__%c:%c%cmain()"
	print_f(s)

if __name__ == "__main__":
	main()

File: C/Colleen/Colleen.c
#include <stdio.h>

/*
	Print function
*/

void print(char *s)
{
	printf(s, 10, 10, 10, 9, 10, 10, 10, 10, 10, 9, 10, 10, 10, 10, 10, 9, 10, 9, 9, 10, 9, 10, 9, 34, s, 34, 10, 9, 10, 9, 10, 10);
}

int main(void)
{
	/*
		Main function
	*/
	char *s="#include <stdio.h>%c%c/*%c%cPrint function%c*/%c%cvoid print(char *s)%c{%c%cprintf(s, 10, 10, 10, 9, 10, 10, 10, 10, 10, 9, 10, 10, 10, 10, 10, 9, 10, 9, 9, 10, 9, 10, 9, 34, s, 34, 10, 9, 10, 9, 10, 10);%c}%c%cint main(void)%c{%c%c/*%c%c%cMain function%c%c*/%c%cchar *s=%c%s%c;%c%cprint(s);%c%creturn (0);%c}%c";
	print(s);
	return (0);
}
